fs/xfstests.py

```diff
@@ -31,7 +31,7 @@
 from avocado.utils import process, build, git, distro, partition
 from avocado.utils import disk, data_structures, pmem
 from avocado.utils import genio
-from avocado.utils.software_manager import SoftwareManager
+from avocado.utils.software_manager.manager import SoftwareManager
```

Comment:
ah @OjaswinM I missed this, as still, our repository is runner compliant and due to this change our CI/CR break, as we are planning to have this change across the repository soon , can you please revert this

Reply:
Ahh alright, done :)